src/components/AgreementSections/Info.js

```diff
@@ -28,7 +28,7 @@ export default class Info extends React.Component {
 
   render() {
     const { agreement } = this.props;
-
+    const agreementIsClosed = (agreement.agreementStatus !== null && (agreement.agreementStatus.value === 'closed'));
```

Comment:
Also, you can make this a bit more succinct as `const agreementIsClosed = get(agreement, 'agreementStatus.value') === statuses.CLOSED;`

Reply:
Bonus, that also saves you from the inevitable crash when `agreementStatus` is merely `undefined` and not `null`.

Reply:
Thank you, I have done so now